js/version30.js

```
@@ -0,0 +1,872 @@
+    // Update button states
+    document.querySelectorAll('.cube-nav button').forEach(btn => {
+        btn.classList.remove('active');
+    });
+
+    switch(face) {
+        case 'front':
+            currentRotation = { x: 0, y: 0 };
+            const frontBtn = document.querySelector('.cube-nav button:nth-child(1)');
+            if (frontBtn) frontBtn.classList.add('active');
+            break;
+        case 'right':
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)